problems/41.Implement Stack using Queues.gs

```diff
@@ -0,0 +1,110 @@
+class MyNode {
+    val: number
+    next: MyNode | null
+    constructor(value: number, nextEle: MyNode | null) {
+        this.val = value
+        this.next = nextEle
+    }
+}
+
+function add(val: number, head: MyNode | null): MyNode {
+    if (head !== null) {
+        let cur = head
+        while (cur.next !== null) {
+            cur = cur.next
+        }
+        cur.next = new MyNode(val, null)
+    } else {
+        head = new MyNode(val, null)
+    }
+    return head
+}
+function print(head: MyNode | null) {
+    let cur = head
+    while (cur != null) {
+        console.log(cur.val)
+        cur = cur.next
+    }
+}
+
+function remove(head: MyNode | null): MyNode | null {
+    if (head == null) {
+        return null
+    }
+    if (head.next == null) {
+        return null
+    }
+    if (head.next.next == null) {
+        head.next = null
+    }
+    if (head != null) {
+        let cur = head
+        if (cur.next != null && cur.next.next != null) {
+            while (cur.next?.next != null) {
```

Comment:
This part of the code looks like mad dag write it. When the hell did you do here?